MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 recursive-exclude niworkflows/ conftest.py
 include versioneer.py
+include niworkflows/_version.py
```

Comment:
Not needed. Should remove it from versioneer so it doesn't keep getting added...

```suggestion
```